recipe/meta.yaml

```diff
@@ -1,36 +1,43 @@
 {% set name = "lockfile" %}
 {% set version = "0.12.2" %}
-{% set hash_type = "sha256" %}
-{% set hash_value = "6aed02de03cba24efabcd600b30540140634fc06cfa603822d508d5361e9f799" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  '{{ hash_type }}': '{{ hash_value }}'
+  sha256: 6aed02de03cba24efabcd600b30540140634fc06cfa603822d508d5361e9f799
 
 build:
-  script: python setup.py install
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
+    - pip
     - python
-    - pbr
+    - pbr >=1.8
+    - setuptools
+    - wheel
   run:
     - python
 
 test:
   imports:
     - lockfile
+  requires:
+    - pip
+  commands:
+    - pip check
```

Comment:
you can add upstream tests from the tarball:
```
    source_files:
      - test
    requires:
      - pytest
    commands
      - pytest test
```

Reply:
I attempted to add as mentioned, but the tests are failing. The upstream tests seem to be pretty outdated and since this particular package has not been maintained for nearly 10 years, I don't think the tests are worth the effort.